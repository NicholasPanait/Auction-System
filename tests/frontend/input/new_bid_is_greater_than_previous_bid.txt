login
testingAdmin
password
bid
itemToBidOn
sellerUserTest
70
logout
