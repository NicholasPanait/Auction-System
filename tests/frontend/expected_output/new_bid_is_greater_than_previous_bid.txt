Welcome to the Online Auction Sales Service!

Please enter login to Login.
login - Login

***LOGIN***
Enter Username: 
Enter Password: 
Welcome Admin User: testingAdmin!

***MENU***
logout - Logout
delete - Delete User Account
create - Create New Account
advertise - Put up item for Auction
bid - Make a bid on an available item
refund - Issue a refund
addcredit - Add Credit
accounts - Display all active accounts
listitems - List Items Up For Auction
changepassword - Change Your Password
***MAKING A BID***

Enter Item Name: 
Enter seller's username: 
Current Highest Bid: 65.00
Enter New Bid: A bid has been successfully made for this item!

***MENU***
logout - Logout
delete - Delete User Account
create - Create New Account
advertise - Put up item for Auction
bid - Make a bid on an available item
refund - Issue a refund
addcredit - Add Credit
accounts - Display all active accounts
listitems - List Items Up For Auction
changepassword - Change Your Password

You have successfully logged out.
